test/test_resize.cpp

```diff
@@ -1122,6 +1122,76 @@ TEST_F(NVFuserTest, FusionResizeSlice5_CUDA) {
   testValidate(&fusion, cg_outputs, aten_inputs, {t2, t4}, __LINE__, __FILE__);
 }
 
+// Test slice with a variety of (constant) inputs
+TEST_F(NVFuserTest, FusionResizeSliceShmoo_CUDA) {
+  for (auto [start, stop] : std::vector<std::pair<int64_t, int64_t>>(
+           {// Slice with end beyond size of input. This should clip to input,
+            // not pad.
+            {0, 11},
+            {11, 13},
```

Comment:
This case blows the stack with infinite recursion trying to simplify index expression. The first case works fine.

Reply:
Sounds like a bug of `SimplifyingIrBuilder`

Reply:
I think it's just due to not handling the `where` expression I implemented. We could probably get it to terminate, but it won't be able to simplify it since we can't prove which branch we've taken unless we concretize differently, in which case we should probably just go ahead and replace the expression with its known simpler form (see above).